n_request.c

```diff
@@ -283,34 +283,47 @@ J *NoteRequestResponseWithRetry(J *req, uint32_t timeoutSeconds)
  @brief Send a request to the Notecard and return the response.
 
  Unlike NoteRequestResponse, this function expects the request to be a valid
- JSON C-string, rather than a `J` object. It also returns the response as a
- dynamically allocated JSON C-string. The caller is responsible for freeing this
- string.
+ JSON C-string, rather than a `J` object. This string may be newline-terminated,
+ but it is not required. The response is returned as a dynamically allocated
+ JSON C-string. The response string is verbatim what was sent by the Notecard.
+ The terminating CR-LF will be included in the response string. The caller is
```

Comment:
What's the rationale for returning the framing delimiter? Not questioning it, but would be good to know in case it comes up in future design discussions.

Reply:
I think the idea is to return exactly what's received from the Notecard, which includes the \n. But I don't think it's critical, personally.